File: C#/BluffinMuffin.Protocol/Lobby/CreateTableResponse.cs
using BluffinMuffin.Protocol.DataTypes.Attributes;

namespace BluffinMuffin.Protocol.Lobby
{
    /// <summary>
    /// 
    /// </summary>
    public class CreateTableResponse : AbstractBluffinReponse<CreateTableCommand>
    {
        /// <summary>
        /// The id of the table that was created
        /// </summary>
        [ExampleValue(42)]
        public int IdTable { get; set; }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public CreateTableResponse(CreateTableCommand command)
            : base(command)
        {
        }
    }
}

File: Python/BluffinMuffin.Protocol/Lobby/CreateTableResponse.py
from AbstractResponse import AbstractResponse
from Lobby.CreateTableCommand import CreateTableCommand

__author__ = 'ericm'
_project_ = 'BluffinMuffin.Protocol'


class CreateTableResponse(AbstractResponse):
    def __init__(self, obj):
        super().__init__('CreateTableResponse', obj, CreateTableCommand(obj['Command']))
        self.id_table = obj['IdTable']

    def __str__( self ):
        return '{0} => {1}'.format(super().__str__(), self.id_table)